tasks/images.js

```diff
@@ -21,9 +21,9 @@ gulp.task('images', callback => {
     runSequence(
         'clean:images',
         ['copy:img', 'copy:assets'],
-        'images:optimise',
+        ...(config.docs.optimiseImages ? ['images:optimise'] : []),
```

Comment:
@Li147 Please can you commit the update to this so that it's `config.img` rather than `config.docs`?

Reply:
done